WPF/Practise_02.01/2020.02.01/MainWindow.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<Window x:Class="_2020._02._01.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:_2020._02._01"
+        xmlns:model="clr-namespace:_2020._02._01.Model"
+        xmlns:viewModel="clr-namespace:_2020._02._01.ViewModel"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800">
+    <Window.Resources>
+        <DataTemplate x:Key="CarTemplate" DataType="{x:Type model:Car}">
+            <StackPanel Orientation="Horizontal">
+                <Image Source="{Binding ImagePath}"
+                       Width="70"
+                       Height="50"></Image>
+                <Label Content="{Binding Brand}"></Label>
+                <Label Content="{Binding Model}"></Label>
+            </StackPanel>
+        </DataTemplate>
+    </Window.Resources>
+    <Window.DataContext>
+        <viewModel:MainWindowViewModel/>
+    </Window.DataContext>
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="27*"/>
+            <ColumnDefinition Width="17*"/>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="331*"/>
+            <RowDefinition Height="88*"/>
+        </Grid.RowDefinitions>
+        <ListView Grid.Column="0"
+                  Grid.Row="0"
+                  ItemsSource="{Binding Cars}"
+                  ItemTemplate="{StaticResource CarTemplate}"
+                  SelectedItem="{Binding SelectedCar}"></ListView>
+        <StackPanel Grid.Column="1"
```

Comment:
According to the task description, the detailed information should have been placed in a separate UserControl.

Reply:
-5 to task.